Update manage-billing-notifications.md (#9108)
MicrosoftDocs/microsoft-365-docs: microsoft-365/commerce/billing-and-payments/manage-billing-notifications.md

To change the primary and alternate email address of other admins in your organization, use the following steps.

> [!NOTE]

> > Billing admins can only change their own alternate email address. They can not modify other admins' email addresses.

Reviewer: Please change to:

Billing admins can only change their own alternate email address. They can't change email addresses for other admins.

> > Billing admins can only change their own alternate email address. They can not modify other admins' email addresses.

Reviewer 2: ```suggestion
> Billing admins can only change their own alternate email address. They can't change email addresses for other admins.
```
